vimrc

```diff
@@ -195,7 +195,9 @@ let g:spec_runner_dispatcher = 'call Send_to_Tmux("clear\n{command}\n")'
 let g:indentobject_meaningful_indentation = ['haml', 'sass', 'yaml', 'markdown']
 let g:indentLine_fileType = ['yaml']
 
-let g:rspec_command = 'call Send_to_Tmux("rspec {spec}\n")'
+if !system("tmate show-messages | grep tmate &> /dev/null")
```

Comment:
Also, is it the error message you're testing against?  I found that $TMUX env var includes a long string, including tmate when it's tmate.  Just an idea to explore that may be a bit more robust depending on how you wanted to use this condition.

That string looks something like this:
`/private/var/folders/g9/9ckx387yx4z_xynh47bbn3n40000gn/T/tmate-501/SLZvIT,3627,0`